data/styles/banner.css

```
@@ -48,41 +48,38 @@
 }
 
 .banner button {
-    background-color: alpha(@banner_fg_color, 0.6);
-    background-image: none;
-    border-color: alpha(@banner_fg_color, 0.7);
-    box-shadow: none;
+    background-color: @banner_fg_color;
+    border: none;
+    box-shadow:
+        inset -1px 0 0 0 alpha(@banner_bg_color, 0.25),
+        inset 1px 0 0 0 alpha(@banner_bg_color, 0.25),
+        inset 0 -1px 0 0 alpha(@banner_bg_color, 0.25),
+        inset 0 1px 0 0 alpha(@banner_bg_color, 0.25);
```

I think it looks better to just not have box shadow tbh
```suggestion
    box-shadow: none;
```